src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -2368,7 +2368,8 @@ public static bool IsLabelContext(this SyntaxTree syntaxTree, int position, Canc
 
             // 1..|
             if (token.IsKind(SyntaxKind.DotDotToken) &&
-                token.Parent.IsKind(SyntaxKind.RangeExpression))
+                token.Parent.IsKind(SyntaxKind.RangeExpression) &&
+                position == token.Span.End)
```

Comment:
what if the position is one or more spaces after the `..`?

Reply:
do you want to use tokenOnLeftOfPosition instead?

Reply:
Replacing `token` with `tokenOnLeftOfPosition` and removing the last condition doesn't help here, since `var token = tokenOnLeftOfPosition.GetPreviousTokenIfTouchingWord(position)`. Considering, that `GetPreviousTokenIfTouchingWord` would return previous token only when the current token is a word and a word is an identifier or keyword, `token` and `tokenOnLeftOfPosition` are always equal here

Reply:
oh.  then my question remains.  how does this work if there is whitespace?

Reply:
It just didn't work. Fixed